docs/release/releasing.md

```diff
@@ -1,19 +1,17 @@
 # Releasing
+1. Ensure that the next version maintains SemVer:
+    * Check if `Unreleased` changes need a MINOR or MAJOR bump.
+    * If yes, then mark the next version as per [`gradle-release` plugin] docs.
+    * Otherwise, it will default to a PATCH bump.
```

Comment:
We could stop using default and make it required in GH actions instead of the `yes/no` param.